.eslintrc.js

```diff
@@ -81,12 +97,16 @@ module.exports = {
         node: true,
         jest: true
       },
-      plugins: ["jest"],
-      extends: ["plugin:jest/recommended"],
+      plugins: ["jest", "testing-library"],
+      extends: ["plugin:jest/recommended", "plugin:testing-library/react"],
       rules: {
-        "@typescript-eslint/no-require-imports": "off",
         "@typescript-eslint/no-non-null-assertion": "off",
-        "jest/no-done-callback": "off"
+        // require() can be useful in mocking
+        "@typescript-eslint/no-require-imports": "off",
+        "@typescript-eslint/no-var-requires": "off",
+        "jest/no-done-callback": "off",
+        // var is useful in mocking due to its hoisting semantics
+        "no-var": "off"
```

Comment:
ESLint tells me there are 57 occurrences in CLUE. I don't know precisely how `var` hoisting interacts with `jest.mock` hoisting, but I do know that `var`s can be referenced in mock implementations while `const`s can't and I've spent time trying alternatives in the past so I'm inclined to keep it this way and explain it to anyone who asks. I wouldn't accept having it affect non-test code, but I think it's reasonable to relax the standards for test-only code.

Reply:
For the record, I replaced all occurrences of `var ` in clue with `const ` (and in one place `let `) and the tests continued to pass. So Kirk removed the disabling of the `var` rule for test files.

The only usage of require in CLUE is to mock the `debug.ts` module. It is uses so some global constants from that module can be modified in a test.  This doesn't seem great, I'd expect it might break at some point.  So I'd still recommend removing the require exception so developers are less inclined to take this kind of approach.  But I won't hold up this PR for that.